cmd/connector/main.go

```diff
@@ -16,14 +16,15 @@ package main
 
 import (
 	sf "github.com/conduitio-labs/conduit-connector-salesforce"
-	sfSource "github.com/conduitio-labs/conduit-connector-salesforce/source"
+	// sfSource "github.com/conduitio-labs/conduit-connector-salesforce/source"
```

Comment:
```suggestion
```
should we remove this line?

Reply:
another thought - is the desire to have these as separate modes for the connector? If so, then we can have a config around it. 

If not, then we should delete the old sfdc source connector code alltogether. It can be done in a separate PR though to limit scope.

Reply:
Im not really sure what to do with the old connector, its still a technically valid way to get data

Reply:
@simonl2002 Do you think we should keep the old source ?

Reply:
@anna-cross yes we should keep the other mode.